firmware/stm32f4_disc/Core/Inc/uart-gps.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * gps.h
+ *
+ *  Created on: Nov 15, 2019
+ *      Author: Bulanov Konstantin
+ */
+
+#define	GPS_USART	&huart4
```

Comment:
Could we change this so we can choose which uart handler to use? We could do something like i2c-rtc.c where the handler is passed into the Init function